packages/wonder-blocks-form/src/components/text-area.tsx

```diff
@@ -377,15 +354,15 @@ const styles = StyleSheet.create({
         },
     },
     error: {
-        background: states.error.background,
-        border: `${border.width.thin} solid ${states.error.border}`,
-        color: states.error.foreground,
+        background: semanticColor.input.error.background,
+        border: `${border.width.thin} solid ${semanticColor.input.error.border}`,
+        color: semanticColor.input.error.foreground,
         "::placeholder": {
             color: semanticColor.text.secondary,
         },
         ":focus-visible": {
             outline: `${border.width.medium} solid ${semanticColor.focus.outer}`,
-            borderColor: states.error.border,
+            borderColor: semanticColor.input.error.border,
```

Comment:
Will this be replaced by the global focus style later on for all the inputs?

Reply:
Yes! in here: https://khanacademy.atlassian.net/browse/WB-1864 (I'll add the TODO comment)